contracts/traits/diamond/mod.rs

```diff
@@ -0,0 +1,86 @@
+pub use crate::traits::{
+    errors::{
+        DiamondError,
+        OwnableError,
+    },
+    ownable::*,
+};
+use brush::traits::Hash;
+use ink_prelude::vec::Vec;
+
+#[brush::wrapper]
+pub type DiamondRef = dyn Diamond;
+
+pub type Selector = [u8; 4];
+
+pub enum FacetCutAction {
```

Comment:
Let's remove that, we can simplify the API. For each new "cut" we will override old selectors. It is not often process so it is okay if the user should specify the full list of expected methods. It will simplify usage and the code of diamond